h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -1119,7 +1119,7 @@ else if(_parms._family == Family.gaussian && _parms._link == Link.identity)
         if(_parms._family != Family.binomial && _parms._family != Family.poisson) {
           seEst = true;
           ComputeSETsk ct = new ComputeSETsk(null, _state.activeData(), _job._key, beta, _parms).doAll(_state.activeData()._adaptedFrame);
-          se = ct._sumsqe / (_nobs - 1 - _state.activeData().fullN());
+          se = ct._sumsqe / (_nobs - 1 - _state.activeData().fullN());  // this is the dispersion parameter estimate
         }
         double [] zvalues = MemoryManager.malloc8d(_state.activeData().fullN()+1);
```

Comment:
I used the dispersion parameter estimate like this from the doc:

![image](https://user-images.githubusercontent.com/7231712/58358832-71da2900-7e35-11e9-9d77-a70a8eebcf49.png)